documentation/decisions/adr/00NN-frontend-testing.md

```diff
@@ -0,0 +1,96 @@
+# Front-end Testing & Coverage
+
+- **Status:** Proposed <!-- REQUIRED -->
+- **Last Modified:** 2023-07-17 <!-- REQUIRED -->
+- **Related Issue:** [183](https://github.com/HHS/grants-equity/issues/183) <!-- RECOMMENDED -->
+- **Deciders:** Sammy Steiner, Lucas Brown, Billy Daly, Andy Cochran <!-- REQUIRED -->
+- **Tags:** ADR <!-- OPTIONAL -->
+
+## Context and Problem Statement
+
+Front-end testing is essential in maintaining a stable and healthy codebase, creating UI interfaces without regression, and an important part of the developer workflow.
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+- **Robust:** Chosen testing frameworks should have features that offer diverse ways of verifying front-end codebase functionality, reducing the need for intensive manual testing
+- **Well-maintained:** Accessible tooling is well-maintained by owners and keeps up with current ecosystems in which it will be integrated.
+- **Ease of use:** Achieving high coverage should be attainable. ICs should be able to onboard with the tooling and execute in a reasonable time frame to maintain that coverage.
+- **Fast:** Running tests shouldn't take ages.
+
+## Options Considered
+
+## Unit Testing
+- Jest
+- Mocha
+
+### Pros and Cons of the Options <!-- OPTIONAL -->
+### [Jest](https://jestjs.io/)
```

Comment:
```suggestion
#### [Jest](https://jestjs.io/)
```
Nit: might be good to use H4 for this to indicate that this is a sub-header of "Options considered"